domain/domain.go

```diff
@@ -2415,26 +2411,11 @@ func (do *Domain) updateStatsWorker(ctx sessionctx.Context, owner owner.Manager)
 			if err != nil {
 				logutil.BgLogger().Debug("dump stats delta failed", zap.Error(err))
 			}
-			statsHandle.UpdateErrorRate(do.InfoSchema())
```

Comment:
Yes.
Besides this, the `type ErrorRate` can also be removed. It's used to decide if we can rely on the stats when estimating. Only feedback updates `ErrorRate`, so it has been useless for a long time. I think it can also be removed now.
@winoros can help to double-check.